server/data/mappers/supportStrategyResponseDtoMapper.ts

```diff
@@ -3,7 +3,7 @@ import type { SupportStrategyListResponse, SupportStrategyResponse } from 'suppo
 import toReferenceAndAuditable from './referencedAndAuditableMapper'
 
 const toSupportStrategyResponseDtos = (apiResponse: SupportStrategyListResponse): SupportStrategyResponseDto[] => {
-  return (apiResponse.supportStrategies as Array<SupportStrategyResponse>).map(toSupportStrategyResponseDto)
+  return ((apiResponse?.supportStrategies || []) as Array<SupportStrategyResponse>).map(toSupportStrategyResponseDto)
```

Comment:
An improvement to cater for the fact that the apiResponse object or the supportStrategies property could be null - in which case we should just return an empty array
(this is what we do with other similar mappers, and will prevent an uncaught error if MN/Curious return a 404/no data for a given prisoner)